EFCore.VisualBasic/Scaffolding/Internal/VisualBasicDbContextGenerator.vb

```diff
@@ -20,7 +21,6 @@ Namespace Scaffolding.Internal
     '''     doing so can result in application failures when updating to a new Entity Framework Core release.
     ''' </summary>
     Public Class VisualBasicDbContextGenerator
```

Comment:
Should we leave the following classes `Public`?
VisualBasicDbContextGenerator
VisualBasicEntityTypeGenerator

Reply:
No need. Anyone who wants to customize the generated code will probably want to use templates instead. (the long-term motivation for #18)